### PR TITLE
Updated generated clients

### PR DESCRIPTION
**Pull request by stevekuznetsov**
hack: add a script to validate generated clients

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

pkg/client: regenerate clients for multi-cluster

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

/assign @ncdc @sttts @imjasonh

### CODE REVIEW

File: hack/verify-codegen-clients.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```
Comment (ncdc) [inline hack/verify-codegen-clients.sh:1]: /usr/bin/env bash please 😄

File: hack/verify-codegen-clients.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+set -o pipefail
+
+"$( dirname "${BASH_SOURCE[0]}")/update-codegen-clients.sh"
+if ! git diff --quiet --exit-code -- pkg/client; then
+	cat << EOF
+ERROR: This check enforces that the client code is generated correctly.
+ERROR: The client code is out of date. Run the following command to re-
+ERROR: generate the clients:
+ERROR: $ hack/update-generated-clients.sh
+ERROR: The following errors were found:
```
Comment (ncdc) [inline hack/verify-codegen-clients.sh:17]:
```suggestion
ERROR: The following differences were found:
```

**Comment by imjasonh** [comment]
lgtm after Andy's comments

Thanks for doing this!

**Comment by stevekuznetsov** [comment]
@ncdc done

**Comment by ncdc** [comment]
Thanks!

**Pull request merged**